sqlstore/src/main/java/uk/ac/bbsrc/tgac/miso/persistence/impl/HibernateInstrumentDao.java

```diff
@@ -142,13 +143,13 @@ public boolean remove(SequencerReference sr) throws IOException {
   }
 
   @Override
-  public Map<String, Integer> getSequencerReferenceColumnSizes() throws IOException {
-    return DbUtils.getColumnSizes(jdbcTemplate, SEQUENCER_REFERENCE_TABLE_NAME);
+  public Map<String, Integer> getInstrumentColumnSizes() throws IOException {
+    return DbUtils.getColumnSizes(jdbcTemplate, INSTRUMENT_TABLE_NAME);
   }
 
   @Override
   public String getFriendlyName() {
-    return "Sequencer";
+    return "Instrument";
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Use already-defined constant 'INSTRUMENT_TABLE_NAME' instead of duplicating its value here. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)